llvm/test/CodeGen/X86/apx/no-rex2-general.ll

```diff
@@ -0,0 +1,81 @@
+; NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py
+; RUN: llc < %s -mtriple=x86_64-unknown -stop-after=x86-isel -mattr=+sse2,+ssse3,+egpr  | FileCheck %s --check-prefix=SSE
+; RUN: llc < %s -mtriple=x86_64-unknown -stop-after=x86-isel -mattr=+sse2,+ssse3,+egpr,+avx | FileCheck %s --check-prefix=AVX
```

Comment:
Should we add tests for fast isel?

Reply:
The API `getRegClass` is shared by all ISEL. I think there is no need to add standalone tests for fast isel.